pages/requester/create-task.tsx

```diff
@@ -28,19 +31,20 @@ export default function Form() {
   const [maxTaskers, setMaxTaskers] = useState(0);
 
   // Errors associated with task overvieww
-  const [titleError, setTitleError] = useState(false);
-  const [descriptionError, setDescriptionError] = useState(false);
-  const [cryptoAllocatedError, setCryptoAllocatedError] = useState(false);
-  const [maxTaskersError, setMaxTaskersError] = useState(false);
+  const [titleError, setTitleError] = useState<boolean>();
```

Comment:
reasoning behind this implementation change? is default still set to false of the boolean type?

Reply:
If you initialize all errors to `false`, that's basically saying that not inputting anything and clicking "Post" is a valid action. The better way is to initialize everything as `undefined`, and then set them to `true` or `false` based on user input. When performing validation, you'd just have to check whether the errors are `undefined` or `false` to know if something's invalid.